cmake/projects/odb-compiler/hunter.cmake

```diff
@@ -29,9 +29,9 @@ elseif(CMAKE_HOST_UNIX)
       VERSION
       "2.4.0"
       URL
-      "http://www.codesynthesis.com/download/odb/2.4/odb-2.4.0-i686-linux-gnu.tar.bz2"
```

Comment:
I would guess that's dependent on the host architecture, you are right. I don't know how to achieve that since the `CMAKE_SIZEOF_VOID_P` is dependent on the target as far as I understood it.